src/Symfony/Component/Messenger/Bridge/Doctrine/Transport/Connection.php

```diff
@@ -46,10 +48,13 @@ class Connection implements ResetInterface
     protected const DEFAULT_OPTIONS = [
         'table_name' => 'messenger_messages',
         'queue_name' => 'default',
+        'batch_size' => 1,
         'redeliver_timeout' => 3600,
         'auto_setup' => true,
     ];
 
+    public const FLAGGED_FOR_DELETION = '9999-12-31 23:59:59';
```

Comment:
To be reverted as not related to this PR.

Reply:
Or at least, make it private.

Reply:
I made it private. 
What about [this one](https://github.com/symfony/symfony/pull/51034/files#r1281056793)?
Is it ok to have that one public, or should I drop that commit? (only ahs that change)

Reply:
I removed this and other small fixes from the PR as to remove noise in the PR